src/components/common/ui/progress-mui/Progress.styles.ts

```diff
@@ -0,0 +1,14 @@
+import { SxProps, Theme } from '@mui/material/styles';
+
+export const progressFront: SxProps<Theme> = {
+  position: 'relative',
```

Comment:
does it do smth?
<img width="143" alt="Screenshot 2023-05-27 at 22 32 38" src="https://github.com/fictadvisor/fictadvisor-web/assets/89708817/f28eb7fc-150a-45a3-a411-8d80ebf01abb">
<img width="177" alt="Screenshot 2023-05-27 at 22 32 30" src="https://github.com/fictadvisor/fictadvisor-web/assets/89708817/e7a56faa-aff3-413a-891c-0542f2c79b58">